docs/pages/versions/unversioned/sdk/location.mdx

```diff
@@ -166,6 +166,13 @@ When using Geofencing, the following platform differences apply:
   20](https://developer.apple.com/documentation/corelocation/monitoring_the_user_s_proximity_to_geographic_regions)
   `regions` that can be simultaneously monitored.
 
+### Background Permissions
+
+To use location tracking or Geofencing in the background, you must request the appropriate permissions:
+
+- On iOS, it must be granted with the `Always` option using `requestBackgroundPermissionsAsync`.
+- On Android, you must request both foreground and background permissions.
```

Comment:
```suggestion
- On Android, you must request both foreground and background permissions.
- On iOS, it must be granted with the `Always` option using [`requestBackgroundPermissionsAsync`](#locationrequestbackgroundpermissionsasync).
```

- Platform order preference as per the writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms

- Let's also add the internal link to the method definition so that it a developer can read more about this method.